src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -47,14 +48,15 @@ enum BuildError {
     },
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone, Default)]
+#[derive(Deserialize, Serialize, Debug, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct CloudwatchLogsSinkConfig {
     pub group_name: Template,
     pub stream_name: Template,
     #[serde(flatten)]
     pub region: RegionOrEndpoint,
-    pub encoding: Encoding,
+    #[serde(deserialize_with = "EncodingConfig::from_deserializer")]
```

Comment:
Leaving this as a note: we should remove this and impl a custom version to avoid having to add this everywhere. We discussed this offline.

Reply:
We fixed this together =D